README.md

```diff
@@ -60,7 +63,17 @@ The toolchain version is managed by the `rust-toolchain.toml` file. If you do no
 
 You will need to install the Protobuf Compiler. This can be done by executing:
 
-`sudo apt install -y protobuf-compiler`
+```shell
+sudo apt install -y protobuf-compiler
+```
+
+### <a name="install-fontconfig-dev-lib">Install fontconfig-dev library</a>
```

Comment:
is it necessary to have the section titles in <a/> tags? I assume this is for the table of contents; what's wrong with linking to the default section name `install-fontconfig-dev-library`? I can see a few other tags which I don't think are necessary and could be removed without changing the table of contents

Reply:
These are fine.  Leaving as is.